.github/workflows/build_container.yml

```diff
@@ -30,6 +30,11 @@ env:
 
 permissions: {}
 
+# Prevent concurrent runs of this workflow, but do not cancel any in-progress
+concurrency:
```

Comment:
how does this end up working? Just queuing up releases?

Reply:
Yep exactly, it'll queue them up in order of PR merges. The reason I did this is so that `latest` is always the last merged commit, otherwise there's a race for which build ends last.